README.md

```diff
@@ -73,7 +73,7 @@ This collection contains modules and plugins to assist in automating [DigitalOce
 
 The collection is tested and supported with:
 
-- ansible-core >= 2.12 (not including `devel`)
+- ansible-core >= 2.14 (not including `devel`)
```

Comment:
Can dropping support for ansible-core 2.12 and 2.13 be split out into a separate change? Dropping support for old ansible-core versions warrants a new major release and isn't really related to the issue at hand.